ivy/functional/frontends/numpy/random/functions.py

```diff
@@ -61,3 +61,13 @@ def multinomial(n, pvals, size=None):
     else:
         num_samples = len(pvals)
     return ivy.multinomial(n, num_samples, batch_size=batch_size, probs=pvals)
+
+
+@to_ivy_arrays_and_back
+@from_zero_dim_arrays_to_scalar
+def permutation(x, /):
+    if isinstance(x, int):
+        x = ivy.arange(x)
+    elif x.ndim > 0:
+        x = x
```

Comment:
we can remove these two lines, as the output is the same whether `x.ndim > 0` or not